modelerfour/test/scenarios/body-array/flattened.yaml

```diff
@@ -341,12 +330,23 @@ schemas: !<!Schemas>
       apiVersions:
       - !<!ApiVersion> 
         version: 1.0.0
-      choiceType: *ref_2
+      choiceType: *ref_1
       language: !<!Languages> 
         default:
-          name: Enum0
+          name: FooEnum
```

Comment:
Interesting that this seems to be swapped with the Enum0 above, what do you think caused that?

Reply:
I think this is an artifact of getting reordered because something changed above. this is just _moved_; not renamed